data/tlg0094/tlg001/tlg0094.tlg001.perseus-eng2.xml

```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://www.stoa.org/epidoc/schema/latest/tei-epidoc.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+<teiHeader xml:lang="eng">
+      <fileDesc>
+         <titleStmt>
+            <title>Of the Names of Rivers and Mountains, and of Such Things as are to be Found Therein</title>
+            <author>Pseudo-Plutarch</author>
+            <editor>William W. Goodwin</editor>
+         	<editor role="translator">R. White</editor>
+         	<sponsor>Perseus Project, Tufts University</sponsor>
+         	<principal>Gregory Crane</principal>
+         	<respStmt>
+         		<resp>Prepared under the supervision of</resp>
+         		<name>Lisa Cerrato</name>
+         		<name>Rashmi Singhal</name>
+         		<name>Bridget Almas</name>
+         	</respStmt>
+         	<funder n="org:NEH" >The National Endowment for the Humanities</funder>
+         </titleStmt>
+      	<publicationStmt>
+      		<publisher>Trustees of Tufts University</publisher>
+      		<pubPlace>Medford, MA</pubPlace>
+      		<authority>Perseus Digital Library Project</authority>
+      		<date type="release" when="2010-12-13">Perseus 4.0</date>
+      		<idno type="filename">tlg0094.tlg001.perseus-eng2.xml</idno>
+      		<availability>
+      <licence target="https://creativecommons.org/licenses/by-sa/4.0/">Available under a Creative Commons Attribution-ShareAlike 4.0 International License</licence>
+      		</availability>
+      	</publicationStmt>
+      	
+      	<sourceDesc>
+      		<biblStruct>
+      			<monogr>
+      				<author>Plutarch</author>
+      				<title>Plutarch's Morals.</title>
```

Comment:
Maybe take out the period after the title

Reply:
I think that was cut/paste from other files. I suspect it is there also.

Reply:
@lcerrato  you are very probably right!